### PR TITLE
Fix to not add superfluous spaces for code (text)

### PR DESCRIPTION
**Pull request by crossjs**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

Ref: https://github.com/syntax-tree/unist/discussions/46#discussioncomment-921609

### CODE REVIEW

**Comment by crossjs** [comment]
_@types/unist@2.0.5_ breaks type coverage

File: test/index.js [hunk 1]
```diff
@@ -1461,8 +1467,8 @@ test('Code text', (t) => {
   )
 
   t.equal(
-    to({type: 'inlineCode', value: 'a\n1. '}),
```
Comment (wooorm) [inline test/index.js:1464]: this test case (s)hould remain the same, right?

**Review by wooorm — COMMENTED** [review]
Nice! On question!

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
sorry for the wait, landed in https://github.com/syntax-tree/mdast-util-to-markdown/releases/tag/1.1.0!

**Pull request merged**